projects/js-packages/publicize-components/src/components/message-box-control/index.tsx

```diff
@@ -3,25 +3,54 @@ import { TextareaControl } from '@wordpress/components';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { useCallback, useRef } from 'react';
 
+export const PLACEHOLDER_TEXT = __(
+	'Write a custom message for your social audience here. This message will override your social post content.',
+	'jetpack-publicize-components'
+);
```

Comment:
This is going to create translation issues if locales are loaded async. So, it's better to have this string inside the component or make it a callable function.

Reply:
Done d29df73